packages/guides-restructured-text/src/RestructuredText/Parser/Productions/ListRule.php

```diff
@@ -88,7 +89,7 @@ public function apply(DocumentParserContext $documentParserContext, CompoundNode
             if ($this->isListItemStart($documentIterator->current())) {
                 $listConfig = $this->getItemConfig($documentIterator->current());
                 $items[] = $this->parseListItem($listConfig, $buffer, $documentParserContext);
-                $buffer = new Buffer();
+                $buffer = new Buffer(unindentStrategy: UnindentStrategy::FIRST);
```

Comment:
Note that `ALL` would work here as well, since the first line is always going to be the least indented one.